source/_integrations/risco.markdown

```diff
@@ -65,7 +65,28 @@ And in the reverse direction:
 | Arm Away | Arm |
 | Arm Home | Partial Arm |
 
-Currently supported plaforms:
+## Services
+
+### Service `risco.bypass_zone`
+
+This service marks a zone as bypassed, so that the alarm isn't triggered when the zone is triggered.
+Note you can only bypass a zone when the partitions it belongs to are disarmed, and it will take effect next time you arm.
+Risco automatically un-bypasses the zone after the alarm is disarmed again.
+
+| Service Data Attribute | Required | Description |
+| ---------------------- | -------- | ----------- |
+| `entity_id`            | yes      | String or list of strings of entity_ids of zones. Use entity_id: all to target all zones. |
+
+### Service `risco.unbypass_zone`
+
+This undoes a zone bypass. You can only unbypass a zone when the partitions it belongs to are disarmed.
+
+| Service Data Attribute | Required | Description |
+| ---------------------- | -------- | ----------- |
+| `entity_id`            | yes      | String or list of strings of entity_ids of zones. Use entity_id: all to target all zones. |
```

Comment:
`async_register_entity_service` is used to register this service, which means `entity_id` is optional (as an area could be targeted alternatively.

```suggestion
| `entity_id`            | no     | String or list of strings of entity_ids of zones. Use entity_id: all to target all zones. |
```